### DISCUSSION TITLE
PK-Sim/MoBi observer

### DISCUSSION THREAD

**Discussion by nguyen-tranbach** [Open-Systems-Pharmacology/Forum#1845] [Models]
Dear community,

I would like to know if it is possible to optimise parameters using observed data for the following Observer :
**Fraction of administered radioactivity** following single IV dose administration
(given the 2 metabolites formed via the same enzyme predefined in the PK-Sim model)

Actually I have radiolabelled data following a single bolus IV dose.

The problem is that the radioactivity mesured originated from the primary molecule and its metabolites, being unable to distinguish one from another.

Thank you

**Comment by StephanSchaller** [comment]
I would say: yes, but I guess it depends on available complementary data (what is known about the compound(s) physchem/adme and specifically metabolism (IVIVE possible) and if cold PK data (parent or metabolite only) is also available?).

**Reply by nguyen-tranbach** [reply]
Great, because the answers are Yes for both your questions.
In this case I should have rephrased my question as:
How to create the observer Fraction of administered radioactivity following single IV dose administration, given the 2 metabolites formed via the same enzyme predefined in the PK-Sim model ?
(I am not sure at all if I did it right)

**Reply by StephanSchaller** [reply]
Not much time right now...
Come to Basel next week 😉: https://www.open-systems-pharmacology.org/conference.html

**Reply by StephanSchaller** [reply]
We have an online training course (free for academia) on how to create observers, here:
https://learn.esqlabs.com/
